src/index.js

```diff
@@ -1,64 +1,35 @@
 import './style.css';
+import display from './modules/toDo_UI.js';
 import Icon from './image/icon.png';
 import arrow from './image/arrow.png';
-import more from './image/more.png';
 
 const icon1 = document.querySelector('#icon');
 icon1.src = Icon;
 const arrowimg = document.querySelector('#arrowimg');
 arrowimg.src = arrow;
 
-const todoList = document.querySelector('#todo-list-wrap');
-const taskLists = [
-  {
-    description: 'Build a To Do List App',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Learn Javascript',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'read',
-    completed: false,
-    index: 2,
-  },
-];
+window.addEventListener('load', () => {
+  const todos = JSON.parse(localStorage.getItem('todos')) || [];
+  const newTodoForm = document.querySelector('#new-todo-form');
 
-const display = () => {
-  taskLists.forEach((todo) => {
-    const todoItem = document.createElement('div');
-    todoItem.classList.add('todo-item');
+  newTodoForm.addEventListener('submit', (e) => {
+    e.preventDefault();
 
-    const label = document.createElement('div');
-    const input = document.createElement('input');
-    const span = document.createElement('span');
-    const content = document.createElement('div');
-    const actions = document.createElement('div');
-    const editImg = document.createElement('img');
+    const todo = {
+      content: e.target.elements.content.value,
+      complet: false,
+      id: todos.length + 1,
+    };
 
-    input.type = 'checkbox';
-    input.checked = todo.completed;
-    input.setAttribute('aria-label', 'checkbox');
-    editImg.src = more;
-    editImg.setAttribute('alt', 'restart todo');
+    // Global variable = todos
+    todos.push(todo);
 
-    label.classList.add('label');
-    span.classList.add('bubble');
-    content.classList.add('todo-content');
-    actions.classList.add('actions');
+    localStorage.setItem('todos', JSON.stringify(todos));
```

Comment:
It would be nicer to add this script to addToDo function and call the addToDo function in ` newTodoForm.addEventListener` as shown below:
```js
const addToDo = () => {
   const todos = JSON.parse(localStorage.getItem('todos')) || [];
   const todo = {
      content: e.target.elements.content.value,
      complet: false,
      id: todos.length + 1,
    };
    todos.push(todo);
    localStorage.setItem('todos', JSON.stringify(todos));
}
```
and the addToDo function should be called in `window.addEventListener`:
```js
window.addEventListener('load', () => {
  const newTodoForm = document.querySelector('#new-todo-form');
  newTodoForm.addEventListener('submit', (e) => {
    e.preventDefault();
    addToDo();
    e.target.reset();
    display();
  });
  display();
});
```